modules/phase_field/test/tests/initial_conditions/PolycrystalVoronoi_fromfile.i

```diff
@@ -0,0 +1,60 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 40
+  ny = 40
+  nz = 0
+  xmax = 250
+  ymax = 250
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[GlobalParams]
+  op_num = 12
+  var_name_base = gr
+[]
+
+[Variables]
+  [./PolycrystalVariables]
+  [../]
+[]
+
+[ICs]
+  [./PolycrystalICs]
+    [./PolycrystalColoringIC]
+      polycrystal_ic_uo = voronoi
+    [../]
+  [../]
+[]
+
+[UserObjects]
+  [./voronoi]
+    type = PolycrystalVoronoi
+    coloring_algorithm = bt
```

Comment:
Is the bt algorithm the right choice for this?  I know it's ok for a small problem but I don't know if it's a good idea to use it in an input file that users might run across and then copy-paste from and wonder why it takes forever when doing a large problem.

Reply:
Yes, you are correct. The bt is slow. I will change it.